ui/packages/consul-ui/app/models/auth-method.js

```diff
@@ -1,5 +1,7 @@
 import Model, { attr } from '@ember-data/model';
 import { or } from '@ember/object/computed';
+import Duration from '@icholy/duration';
```

Comment:
Did you figure out what made the golang_compatible branch of this package golang compatible?

Reply:
no, it looks the same in both branches. The master branch does have newer updates.

Reply:
Hmm weird 🤷 

I'm kinda wondering whether we should use this one instead of Duration.js?

https://github.com/jkroso/parse-duration

Seems to be far more well used that Duration.js (100,000s of weekly downloads vs 1,000s) and slightly more featureful and simpler, what do you think?

It'd be a tiny switch the parser inside your computed property.

Reply:
Updated to use `parse-duration`.